backend/mora/service/configuration_options.py

```diff
@@ -0,0 +1,175 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# import sqlite3
+import psycopg2
+import flask
+from mora import exceptions
+from .. import settings
+from .. import util
+
+
+blueprint = flask.Blueprint('configuration', __name__, static_url_path='',
+                            url_prefix='/service')
+
+
+def _get_connection():
+    conn = psycopg2.connect(user=settings.USER_SETTINGS_DB_USER,
+                            dbname=settings.USER_SETTINGS_DB_NAME,
+                            host=settings.USER_SETTINGS_DB_HOST,
+                            port=settings.USER_SETTINGS_DB_PORT,
+                            password=settings.USER_SETTINGS_DB_PASSWORD)
+    return conn
+
+
+@blueprint.route('/ou/<uuid:unitid>/configuration', methods=['POST'])
+@util.restrictargs()
+def set_org_unit_configuration(unitid):
+    """Set a configuration setting for an ou.
+
+    .. :quickref: Unit; Create configuration setting.
+
+    :statuscode 201: Setting created.
+    :statuscode 404: No such unit found.
+
+    :param unitid: The UUID of the organisational unit to be configured.
+
+    :<json object conf: Configuration option
+
+    .. sourcecode:: json
+
+      {
+        "org_units": {
+          "show_location": "True"
+        }
+      }
+
+    :returns: True
+    """
+    conn = _get_connection()
```

Comment:
Please consider using a context manager to guarantee release of the connection and cursor.